gulpfile.js

```diff
@@ -0,0 +1,47 @@
+let gulp = require("gulp");
+let ts = require("gulp-typescript");
+let tsProject = ts.createProject("tsconfig.json");
+let browserify = require("browserify");
+let source = require('vinyl-source-stream');
+let sourcemaps = require('gulp-sourcemaps');
+let merge = require('merge2');
+let tsify = require("tsify");
+let babelify = require("babelify");
+let uglify = require("gulp-uglifyes");
+let buffer = require("vinyl-buffer");
+let paths = {
+  pages: ['src/views/*.html']
+};
+let compile = [
+  'index.ts',
+  './src/**/*.ts',
+  './test/**/*.ts',
+  './src/**/*Spec.ts',
+  '!./src/**/trash/**'
+];
+
+gulp.task("copy-html", function () {
```

Comment:
what is the purpose of copy-html target? may be a rename is required or we can just remove it.

Reply:
Sure. i will remove it